packages/polaris-viz/src/components/FunnelChartNext/components/FunnelChartLabels/FunnelChartLabels.tsx

```diff
@@ -62,7 +69,7 @@ export function FunnelChartLabels({
         const isLast = index === labels.length - 1;
 
         const targetWidth = isLast
-          ? barWidth - GROUP_OFFSET * 3
+          ? barWidth - GROUP_OFFSET
```

Comment:
This might not have been introduces here but why are we multiplying by three when is not the last index? can we create a constant for that value that explicitly tells us its meaning?